doc/source/notebooks/tailor/external-mean-function.pct.py

```diff
@@ -66,32 +75,66 @@ def generate_data(num_functions=10, N=1000):
 # %%
 _ = plt.plot(Xs, F)
 
-
 # %% [markdown]
 # We generate the meta and test tasks.
 
+
 # %%
-def generate_meta_and_test_tasks(num_datapoints, num_meta, num_test):
-    N = 1000
+def generate_meta_and_test_tasks(num_datapoints, num_meta=1000, num_test=200, N=50):
```

Comment:
```suggestion
def generate_meta_and_test_tasks(num_datapoints, num_meta, num_test, N):
```
as you pass in all arguments explicitly anyways it's safer not to risk having some inconsistency between them.